cmd/podman/images.go

```diff
@@ -284,8 +284,11 @@ func getImagesTemplateOutput(ctx context.Context, runtime *libpod.Runtime, image
 		for repo, tags := range image.ReposToMap(img.Names()) {
 			for _, tag := range tags {
 				size, err := img.Size(ctx)
+				var sizeStr string
 				if err != nil {
-					size = nil
+					sizeStr = err.Error()
```

Comment:
Suggest wrapping the error with a little verbiage, but will let @mheon make the call.

Reply:
Probably safe to use the error directly. My only concern would be if it gets to be extremely long and interrupts formatting.